doc/classes/OS.xml

```diff
@@ -153,6 +153,13 @@
 				Returns the keycode of the given string (e.g. "Escape").
 			</description>
 		</method>
+		<method name="get_architecture_name" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the name of the CPU architecture the Godot binary was built for. Possible return values are [code]x86_64[/code], [code]x86_32[/code], [code]arm64[/code], [code]armv7[/code], [code]rv64[/code], [code]riscv[/code], [code]ppc64[/code], [code]ppc[/code], [code]wasm64[/code] and [code]wasm32[/code].
+				[b]Note:[/b] [method get_architecture_name] does [i]not[/i] return the name of the host CPU architecture. For example, if running an x86_32 Godot binary on a x86_64 system, the returned value will be [code]x86_32[/code].
```

Comment:
I think this might make it a candidate for `Engine` and not `OS`. `OS` is for interaction with the OS, `Engine` is for interaction with the Godot build, IMO.